64bit Version

barkink opened the pull request:
Working on all Windows Xp, 2003, 2008, 7 and 8 64bit versions

Commit:
64Bit version added and also executable included

rsmudge commented:
sorry to be slow on the feedback. Can you fix the whitespace in this commit and make the code style reflect the existing loader.c file.

773517913 commented:
Hello @barkink , I've just removed the whitespaces and made the code style similar to existing 32 bit version. Please update your commit.
[main64.zip](https://github.com/rsmudge/metasploit-loader/files/1238757/main64.zip)

barkink committed:
Update main64.c

removed the whitespaces and made the code style similar to existing 32 bit version by @773517913

barkink commented:
@773517913 thanks mate. I updated the commit. @rsmudge can you please approve the pull request?

Thanks.
